<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connect</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>461</width>
        <height>71</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <widget class="QLabel" name="label_FolderSQLITE">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>424</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;PASO 1&lt;/span&gt;&lt;span style=&quot; color:#5555ff;&quot;&gt;:&lt;/span&gt;&lt;span style=&quot; color:#0000ff;&quot;&gt; ELEGIR EL DIRECTORIO DONDE GUARDAR LA BASE DE DATOS DE PROYECTOS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="leFolderSQLITE">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>40</y>
         <width>371</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QToolButton" name="pbnSearchFolder">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>61</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>10</y>
        <width>241</width>
        <height>341</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>90</y>
         <width>221</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_Name">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>78</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="leName">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>70</y>
         <width>130</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_Password">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>280</y>
         <width>78</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_CREDENTIALS">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>235</y>
         <width>75</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>CREDENTIALS :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lePassword">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>280</y>
         <width>130</width>
         <height>20</height>
        </rect>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
      <widget class="QLineEdit" name="leHost">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>174</y>
         <width>130</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_DATABASE">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>59</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>DATABASE :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lePort">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>200</y>
         <width>130</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="leUserName">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>254</y>
         <width>130</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_DBType">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>148</y>
         <width>78</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>DataBase TYPE:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="leDBType">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>148</y>
         <width>130</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_DBName">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>122</y>
         <width>78</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Database Name</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_Host">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>174</y>
         <width>78</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Host</string>
       </property>
      </widget>
      <widget class="Line" name="line_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>226</y>
         <width>211</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_User">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>254</y>
         <width>78</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Username</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="leDBName">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>122</y>
         <width>130</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_Port">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>78</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_PROYECTO">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>201</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>EMPRESA - PROYECTO ELEGIDO:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pbnGuardarProyecto">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>310</y>
         <width>221</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Guardar Proyecto</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_FolderSQLITE_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>221</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;PASO 4&lt;/span&gt;&lt;span style=&quot; color:#0000ff;&quot;&gt;: COMPROBAR DATOS DE CONEXIÃ“N:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pbnProyecto_Ejemplo">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>221</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Proyecto de ejemplo :</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>471</width>
        <height>261</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <widget class="QPushButton" name="pbnMostrarProyectos">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>451</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Mostrar Proyectos Guardados en la Tabla</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="tableWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>451</width>
         <height>141</height>
        </rect>
       </property>
       <row>
        <property name="text">
         <string>New Row</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>New Column</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>1</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>2</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>3</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>4</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>5</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>6</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>8</string>
        </property>
       </column>
      </widget>
      <widget class="QLabel" name="label_FolderSQLITE_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>461</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;PASO 3:&lt;/span&gt;&lt;span style=&quot; color:#0000ff;&quot;&gt; ELEGIR LA FILA CON LA EMPRESA-PROYECTO Y DAR AL BOTÃ“N &amp;quot;CARGAR PROYECTO&amp;quot;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_FolderSQLITE_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>424</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;PASO 2:&lt;/span&gt;&lt;span style=&quot; color:#0000ff;&quot;&gt; MOSTRAR LAS EMPRESAS-PROYECTOS GUARDADOS:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_Fila">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>230</y>
         <width>78</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Fila elegida :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="leFila">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>230</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pbnCargarDatos">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>230</y>
         <width>151</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Cargar Proyecto</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pbnBorrarProyecto">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>230</y>
         <width>161</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Borrar Proyecto</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_FolderSQLITE_3">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>360</y>
        <width>511</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;PASO 5&lt;/span&gt;&lt;span style=&quot; color:#0000ff;&quot;&gt;: CONECTAR PINEBOO A LA EMPRESA-PROYECTO ELEGIDO / ACTIVO pulsando el botÃ³n:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <zorder>groupBox_2</zorder>
     <zorder>groupBox</zorder>
     <zorder>groupBox_3</zorder>
     <zorder>label_FolderSQLITE_3</zorder>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbnStart">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>35</height>
      </size>
     </property>
     <property name="text">
      <string>Start Pineboo</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
